cartridges/app_onboarding/cartridge/models/productListItem.js

```diff
@@ -0,0 +1,30 @@
+'use strict'
```

Comment:
Avoid logic in models.

Especially avoid transactions since it will cause DB call every time this model would be used. It will cause performance issues